src/dissect/dissect.c

```diff
@@ -140,6 +145,12 @@ static int parse_argv(int argc, char *argv[]) {
                                 return r;
                         break;
 
+                case ARG_ROOT_HASH_SIG:
+                        if (free_and_strdup(&arg_root_hash_sig, optarg) < 0)
```

Comment:
parse_path_argument_and_warn(), like a few lines above...

(but as mentioned, maybe we should allow literal signature specification here too? not sure how precisely pkcs7 signature files look like)